src/styles/topbar.css

```diff
@@ -0,0 +1,50 @@
+.topbar {
+  width: 100%;
+  height: 380px;
+  position: sticky;
+  top: 0;
+  z-index: 999; }
+
+.topbarWrapper {
+  background-color: aquamarine;
+  height: 100%;
+  padding: 0px 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; }
+
+.h3 {
+  font-weight: 600;
+}
+
+.logo {
+  font-weight: bold;
+  font-size: 30px;
+  color: #30c68a;
```

Comment:
Note: Tailwind seems to have a cool approach around colors and customising them. 
[See here](https://tailwindcss.com/docs/text-color)